src/transformers/quantizers/base.py

```diff
@@ -99,6 +110,16 @@ def adjust_target_dtype(self, torch_dtype: "torch.dtype") -> "torch.dtype":
         """
         return torch_dtype
 
+    def update_missing_keys(self, model, missing_keys: List[str], prefix: str) -> List[str]:
+        """
+        Override this method if you want to adjust the `missing_keys`.
+
+        Args:
+            missing_keys (`List[str]`, *optional*):
+                The list of missing keys in the checkpoint compared to the state dict of the model
+        """
+        return missing_keys
```

Comment:
Could you explain why we need this, only quanto requires it vs the other schemes that do not -> quanto is not consistent with other saving formats? (naive q on my side!)

Reply:
This is linked to @dacorvo answer below : 
> @SunMarc correct me if I am wrong but I think that is because when activations are quantized, quanto adds two buffers to quantized modules to store the input and output quantization scales (for weights also, but these are stored directly inside the Tensor subclass with the same name, so it is transparent).

So, after converting replacing the layers with the quantized layers, we have two additional buffers for each weight that are missing from in the checkpoint.